src/main/groovy/org/mockito/release/notes/contributors/ContributorsMap.java

```diff
@@ -0,0 +1,8 @@
+package org.mockito.release.notes.contributors;
+
+import org.mockito.release.notes.model.Contributor;
+
+import java.util.Map;
+
+public interface ContributorsMap extends Map<String, Contributor> {
```

Comment:
Fixed. Used `ContributorsSet` class to avoid confusions. Suggested `ContributorProvider` is to similar to Contributor**S**Provider class.